.github/workflows/linux.yaml

```diff
@@ -6,13 +6,18 @@ on:
     # branches: [ master ]
 
 jobs:
-  build_libebml:
-    name: libebml
-    runs-on: ubuntu-latest
+  Linux-GCC:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        cxx: ['7', '13']

```

Comment:
We already have a target to build with gcc-13.

Reply:
Sure, this is a draft PR to see if it fails. This can probably replace the other one in the long run.

Reply:
BTW in VLC we're probably going to move to C17, which means gcc 7 won't be supported. https://code.videolan.org/videolan/vlc/-/merge_requests/4858/diffs#9f621eb5fd3bcb2fa5c7bd228c9b1ad42edc46c8

I don't know if @mbunkus plans to support gcc 7 is some distros for the new libebml/libmatroska.

Reply:
I don't know if that's relevant to a library.

In any case, GCC7 is the earliest supportable version as it has enough C++17 support. I don't think it makes sense to downgrade to C++14  in 2024.

It's also not too difficult to keep support for it so might as well.

Reply:
I does seem to cost to support it since it's exhibiting errors in the compiler that we have to workaround in perfectly fine code.

Reply:
I mean...it's just uninlining one constructor. It's not that much.

Reply:
For now, but as we progress into using more C++17 we might find new things that old GCC doesn't support well.

Reply:
Sure, and that bridge can be crossed then.